rainmaker_orchestrator/server.py

```
@@ -1,426 +1,714 @@
-    return jsonify({
-        "error": "Internal server error",
-        "status": "error"
-    }), 500
-
-
-def cleanup_orchestrator():
+    try:
+        full_path = Path(settings.workspace_path) / file_path
+
+        if not full_path.exists():
+            raise HTTPException(status_code=404, detail="File not found")
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)